### ISSUE TITLE
WiFi setup endpoint should only return array of SSIDs instead of irrelevant details

### ISSUE THREAD

**Issue by larsblumberg** [getsenic/nuimo-hub-backend#27]
Right now `/-/setup/wifi` returns:

```
{"": {"device": "wlan0"}, "BB_Gast": {"device": "wlan0"}, "SETUP": {"device": "wlan0"}, "Firma": {"device": "wlan0"}, "WLAN-B22E17": {"device": "wlan0"}, "gal-net 2.4 GHz": {"device": "wlan0"}, "cwtwlan": {"device": "wlan0"}, "GoodHood": {"device": "wlan0"}, "Bonack-Gast": {"device": "wlan0"}, "senic-sandbox": {"device": "wlan0"}, "senic-net 2.4 GHz": {"device": "wlan0"}, "Ingenius+": {"device": "wlan0"}, "Booming-Games": {"device": "wlan0"}, "nordwind": {"device": "wlan0"}}
```

Instead it should return:

```
["SSID1", "SSID"", ...]
```

**Comment by tomster** [comment]
Implemented in #38

**Comment by tomster** [comment]
This has been merged already by @larsblumberg -> closing
